instrumentation/powerjob-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/powerjob/v4_0/PowerJobProcessRequest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.powerjob.v4_0;
+
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.BASIC_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.BROADCAST_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.DYNAMIC_DATASOURCE_SQL_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.FILE_CLEANUP_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.HTTP_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.MAP_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.MAP_REDUCE_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.PYTHON_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.SHELL_PROCESSOR;
+import static io.opentelemetry.javaagent.instrumentation.powerjob.v4_0.PowerJobConstants.SPRING_DATASOURCE_SQL_PROCESSOR;
+
+import tech.powerjob.official.processors.impl.FileCleanupProcessor;
+import tech.powerjob.official.processors.impl.HttpProcessor;
+import tech.powerjob.official.processors.impl.script.PythonProcessor;
+import tech.powerjob.official.processors.impl.script.ShellProcessor;
+import tech.powerjob.official.processors.impl.sql.DynamicDatasourceSqlProcessor;
+import tech.powerjob.official.processors.impl.sql.SpringDatasourceSqlProcessor;
+import tech.powerjob.worker.core.processor.sdk.BasicProcessor;
+import tech.powerjob.worker.core.processor.sdk.BroadcastProcessor;
+import tech.powerjob.worker.core.processor.sdk.MapProcessor;
+import tech.powerjob.worker.core.processor.sdk.MapReduceProcessor;
+
+public final class PowerJobProcessRequest {
+  private String methodName;
+  private final Long jobId;
+  private String jobType;
+  private Class<?> declaringClass;
+
+  private boolean failed;
```

Comment:
Suggest to delete extra blank above and below this line.

Reply:
@steverao I'd recommend not worrying too much about formatting, since we have spotless which does a pretty decent job